common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -534,12 +533,22 @@ def test_accordion_events(self):
                     'event': {
                         'target_name': 'Test Subsection 2,1 ',
                         'widget_placement': 'accordion',
+
                     }
                 },
             ],
-            outline_ui_events
+            selected_events
         )
 
+    def test_link_clicked_events(self):
+        self.course_nav.go_to_section('Test Section 1', 'Test Subsection 1,2')
+        self.course_nav.go_to_section('Test Section 2', 'Test Subsection 2,1')
+
+        filter_link_clicked = lambda event: event.get('name', '') == 'edx.ui.lms.link_clicked'
+        link_clicked_events = self.wait_for_events(event_filter=filter_link_clicked, timeout=2)
+
+        self.assertEqual(len(link_clicked_events), 2)
```

Comment:
You should be able to test partial contents of the event.  Check the event assertion methods on the superclass for details.  (Failing that, you could manually `assertTrue(event['target_url'].startswith('http://...'))`).